### PR TITLE
config: add wal bytes per sync

### PR DESCRIPTION
**Pull request by siddontang**
Use ycsb `tidb-ycsb -concurrency 200 -workload=F -duration=10m  raw://172.16.20.1:2379`, we can find that using`wal_bytes_per_sync` can improve the 3% QPS if sync = false, but have no big effect if sync = true.

We need to do more benchmark to ensure a suitable value, so here I just add the configuration.
For `bytes_per_sync`, we can use rate limiter which sets `bytes_per_sync` to 1 MB by default.

 ||Raft 0 MB|Raft 64 KB|Raft 1 MB|
 |-|-|-|-|
 |KV 1 MB + Sync false|32038.3|32880.7|32938.9|
 |KV 0 MB + Sync false|31533.7|32876.3|32742.1|
 |KV 0 MB + Sync true|25017.8|24993.2|24492.9|

/cc @zhangjinpeng1987 @huachaohuang

### CODE REVIEW

**Comment by huachaohuang** [comment]
You need to update the etc template and the test template too.

**Comment by siddontang** [comment]
PTAL @huachaohuang @zhangjinpeng1987

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @huachaohuang

File: etc/config-template.toml [hunk 1]
```diff
@@ -231,6 +231,9 @@
 # set backup path, if not set, use "backup" under store path.
 # backup-dir = "/tmp/tikv/store/backup"
 
+# Allows OS to incrementally sync WAL to disk while it is being written.
+# wal-bytes-per-sync = "1MB"
```
Comment (huachaohuang) [inline etc/config-template.toml]: This is not consistent with the default value in the code.

**Comment by siddontang** [comment]
PTAL @huachaohuang

**Pull request merged**